src/apps/reservation-list/list/list.tsx

```diff
@@ -81,7 +81,7 @@ const List: FC<ListProps> = ({
               </>
             </h2>
           </div>
-          <div className="list-reservation-container m-32">
+          <div data-cy="empty-list" className="list-reservation-container m-32">
```

Comment:
Isn't it possible to make  the `EmptyList` component take care of setting the data-cy on itself? That is normally how we have handled cases like this.

Reply:
Done